src/services/resourceQueue.js

```diff
@@ -5,7 +5,7 @@ import { auditLogger } from '../logger';
 const resourceQueue = newQueue('resource');
 
 const addGetResourceMetadataToQueue = (id, url) => {
-  const retries = process.env.FILE_SCAN_RETRIES || 5;
```

Comment:
Might be nice to leave this as a variable (although probably need a different name for this and the one below, instead of FILE_SCAN)

Reply:
This has been updated for both retries and delay.